### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by elpie89**
modified bat file to use env_variable instead of absolute path

### CODE REVIEW

File: 3ds Max/Max2Babylon/2015/Max2Babylon2015.csproj [hunk 1]
```diff
@@ -43,7 +43,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>C:\Program Files\Autodesk\3ds Max 2016\bin\assemblies\</OutputPath>
+    <OutputPath>..\assemblies\2015\</OutputPath>
```
Comment (Drigax) [inline 3ds Max/Max2Babylon/2015/Max2Babylon2015.csproj]: Do we need to place all the binaries in ..\Assemblies\<version>? Why not just change the PostBuild script to retrieve the binaries from their respective bin/<build_flavor> directory?

Comment (Drigax) [reply 3ds Max/Max2Babylon/2015/Max2Babylon2015.csproj]: Sorry to hold up the PR, I just noticed this when testing the installer .bat

Comment (elpie89) [reply 3ds Max/Max2Babylon/2015/Max2Babylon2015.csproj]: I'm not sure if this is what you mean.
I used the default bin/ConfigurationName folder, then the post-build look for the right folder.

Comment (Drigax) [reply 3ds Max/Max2Babylon/2015/Max2Babylon2015.csproj]: I'm alright with using the default build location in bin/<build flavor>, or we can pass $(OutDir) as a command line parameter to the postbuild script.

File: 3ds Max/Max2Babylon/OnPostBuild.bat [hunk 2]
```diff
@@ -5,7 +5,7 @@ SETLOCAL enabledelayedexpansion
 SET max_version=%2
 SET max_location=!ADSK_3DSMAX_x64_%max_version%!
 
-SET source_dir="%~dp0assemblies\%max_version%\"
+SET source_dir="%~dp0%max_version%\bin\%1"
```
Comment (Drigax) [inline 3ds Max/Max2Babylon/OnPostBuild.bat]: also, how are variables like %~dp0 and %~dp0assemblies defined? asking to learn, as I'm totally unfamiliar with this convention? :+1:

Comment (bghgary) [reply 3ds Max/Max2Babylon/OnPostBuild.bat]: Try running "help call" or "call /?" in a command prompt. That will explain what this syntax means. Specifically, in this case, `%~dp0` just means the path of the currently running batch script.

**Comment by Drigax** [comment]
Hey @elpie89 I just want to let you and Luca know how much I appreciate this improvement to our debugging toolkit. You guys save me so many hours of copying files over by hand to test changes.

On top of that thanks in general for your contributions to our exporter toolchain. These tools get better and more capable with each PR 😁

**Comment by elpie89** [comment]
Thanks, @Drigax, glad to hear this.
Actually, I'm Luca and @Selmar is my colleague, he introduced me to all of this.
His support and yours are fundamental for me

**Pull request merged**